api/design/status.go

```diff
@@ -23,12 +23,7 @@ var _ = Service("status", func() {
 
 	Method("Status", func() {
 		Description("Return status 'ok' when the server has started successfully")
-		Result(func() {
-			Attribute("status", String, "Status of server", func() {
-				Example("status", "ok")
-			})
-			Required("status")
-		})
+		Result((server))
```

Comment:
you can direct use `ArrayOf(Service)`... no need to define server I guess

Reply:
Yes I could use `ArrayOf(Service)` but that could give the response as 

```
[
         {
		"name": "api",
		"status": "ok"
	},
	{
		"name": "db",
		"status": "ok"
	}
]
```

and not as 

```
{
	"services": [
		{
			"name": "api",
			"status": "ok"
		},
		{
			"name": "db",
			"status": "ok"
		}
	]
}

```

Reply:
```
Attribute("services", ArrayOf("Service"), "Description")
```
won't this work inside Result?